pkg/secretsencrypt/controller.go

```diff
@@ -217,7 +220,7 @@ func (h *handler) validateReencryptStage(node *corev1.Node, annotation string) (
 
 func (h *handler) updateSecrets(nodeRef *corev1.ObjectReference) error {
 	secretPager := pager.New(pager.SimplePageFunc(func(opts metav1.ListOptions) (runtime.Object, error) {
-		return h.secrets.List(metav1.NamespaceAll, opts)
+		return h.cclient.Secrets(metav1.NamespaceAll).List(h.ctx, metav1.ListOptions{})
```

Comment:
I think you need to leave this passing in `opts`. If you pass in a new listoptions then the pagination stuff from the helper is dropped.